src/qibocal/cli/auto_builder.py

```diff
@@ -0,0 +1,78 @@
+import datetime
+import os
+
+import yaml
+
+from qibocal.auto.execute import Executor
+
+from .builders import ActionBuilder, load_yaml
+
+
+class AutoCalibrationBuilder(ActionBuilder):
+    def __init__(self, runcard, folder=None, force=False):
+        super().__init__(runcard, folder, force)
+        self.executor = Executor.load(self.runcard)
+
+    def run(self):
+        if self.platform is not None:
+            self.platform.connect()
+            self.platform.setup()
+            self.platform.start()
+
+        self.executor.run(self.platform, self.qubits, self.folder)
+
+        if self.platform is not None:
+            self.platform.stop()
+            self.platform.disconnect()
+
+    def dump_report(self):
+        from qibocal.web.report import create_autocalibration_report
+
+        # update end time
+        meta = load_yaml(f"{self.folder}/meta.yml")
```

Comment:
See the other comments:
```suggestion
        meta = yaml.safe_load((self.folder / META).read_text())
```